### ISSUE TITLE
Unable to sort mutliple pages of services by duration down

### ISSUE THREAD

**Issue by dairefagan** [NagiosEnterprises/nagioscore#42]
So we have multiple pages of services flagged with problems, mainly APs and modems etc, and I would like to work from the longest duration flagged towards to shortest. If I click on problems under the current status menu on the left hand side, although I can sort the first page by duration ascending, if I then go on to the second page the sort order is reset, and if I try to counter this by selecting sort by duration ascending I am brought back to the first page. Further, if while on the first page I select order by duration ascending and then select limit results to all or say 250, the sort order is also reset. The same happens if I limit the number first and then select sort by duration ascending.

Basically I can sort by duration ascending, but only the first page.

Please address this, thanks.

**Comment by tmcnag** [comment]
Link back to forum thread for reference:

http://support.nagios.com/forum/viewtopic.php?f=7&t=32954

**Comment by dairefagan** [comment]
Can you please assign this to somebody?

**Comment by tmcnag** [comment]
Unfortunately I don't have that access as I am not a maintainer.  I will see about getting this assigned, however.  I'm not a C dev by trade but I know enough to be useful, so if I find time I will look at this myself.

**Comment by dairefagan** [comment]
Thanks, please update if there is one.
